WcaOnRails/app/assets/stylesheets/users.scss

```diff
@@ -19,3 +19,16 @@ input[type="radio"].center-vertically {
   vertical-align: middle;
   height: 100%;
 }
+
+.input-group.date.datetimepicker {
+  width: 100%;
+}
+
+.form-group {
```

Comment:
Oh, I'll try that.
Em 08/02/2016 17:34, "Jeremy Fleischman" <notifications@github.com>
escreveu:

> In WcaOnRails/app/assets/stylesheets/users.scss
> <https://github.com/cubing/worldcubeassociation.org/pull/379#discussion_r52196315>
> :
>
> > @@ -19,3 +19,16 @@ input[type="radio"].center-vertically {
> >    vertical-align: middle;
> >    height: 100%;
> >  }
> > +
> > +.input-group.date.datetimepicker {
> > +  width: 100%;
> > +}
> > +
> > +.form-group {
>
> All you need to get this special styling is to be a .form-control, right?
> See http://getbootstrap.com/css/#forms-example.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cubing/worldcubeassociation.org/pull/379/files#r52196315>
> .
>